<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="media_windows">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>540</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>541</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Button_start">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_pause">
       <property name="text">
        <string>pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_load">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>360</y>
      <width>551</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="files_list">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>491</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_line">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>50</y>
      <width>31</width>
      <height>311</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="massage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
